builder/src/build/patches/investigated/update-webpack-chunks-file/get-updated-webpack-chunks-file-content.ts

```diff
@@ -0,0 +1,22 @@
+import * as ts from "ts-morph";
+
+import { getChunkInstallationIdentifiers } from "./get-chunk-installation-identifiers";
+import { getFileContentWithUpdatedWebpackFRequireCode } from "./get-file-content-with-updated-webpack-require-f-code";
```

Comment:
`FRequire` and `require-f` - did you mean for these to be in different orders? Do they mean different things?

Reply:
yeah.... I switched the two at some point, and I must have missed to update the file name, thanks for noticing it! 🙏